requirements/production.txt

```diff
@@ -1,5 +1,5 @@
 -r base.txt
 
 gunicorn==20.0.4
-zappa==0.46.1
+zappa==0.53.0
```

Comment:
@symroe had some reservations about this upgrade - but I suppose we can try a stage deploy and downgrade if we need to?